mdpow/fep.py

```diff
@@ -953,6 +994,88 @@ def analyze(self, force=False, stride=None, autosave=True, ncorrel=25000):
         self.logger_DeltaA0()
         return self.results.DeltaA.Gibbs
 
+    def collect_alchemlyb(self, autosave=True, autocompress=True):
+        if self.method in ['TI', 'BAR', 'MBAR']:
+            extract = self.estimators[self.method]['extract']
+        else:
+            errmsg = "The method is not supported."
```

Comment:
```suggestion
            errmsg = "The method %s is not supported." % self.method
```
Where do you set the method for the class? On `__init__`?

Reply:
Yes. And I think we should set the option in the script to choose whether to use alchemlyb.

Reply:
Ok, we'll figure that one out in #133